packages/terra-application-links/src/tabs/ApplicationTabs.module.scss

```diff
@@ -5,8 +5,9 @@
     flex-wrap: nowrap;
     height: 100%;
     justify-content: center;
-    padding-left: var(--terra-application-tabs-spacer-width, 2rem);
-    padding-right: var(--terra-application-tabs-spacer-width, 2rem);
+    overflow: hidden;
+    padding-left: var(--terra-application-links-tabs-container-padding-left, 1.4285714286rem);
+    padding-right: var(--terra-application-links-tabs-container-padding-right, 1.4285714286rem);
```

Comment:
There's a `.tabs-container` block below this block. There will be a collision if that block ever needs a left / right padding.

```scss
  .application-tabs {
    padding-left: var(--terra-application-links-tabs-container-padding-left, 1.4285714286rem);
    padding-right: var(--terra-application-links-tabs-container-padding-right, 1.4285714286rem);
  }

  .tabs-container {
    width: calc(100% - var(--terra-application-links-tabs-container-padding-left, 1.4285714286rem) - var(--terra-application-links-tabs-container-padding-right, 1.4285714286rem));
  }
```

Reply:
Suggest updating `application-tabs` to `tabs-wrapper` so we can do something like this.

```scss
  .tabs-wrapper {
    padding-left: var(--terra-application-links-tabs-wrapper-padding-left, 1.4285714286rem);
    padding-right: var(--terra-application-links-tabs-wrapper-padding-right, 1.4285714286rem);
  }

  .tabs-container {
    width: calc(100% - var(--terra-application-links-tabs-container-padding-left, 1.4285714286rem) - var(--terra-application-links-tabs-container-padding-right, 1.4285714286rem));
  }
```

Reply:
Updated here https://github.com/cerner/terra-framework/pull/439/commits/ca6e18fc82385d7f2349879e6f35f25f1f502ed0